undertow/src/main/java/org/wildfly/extension/undertow/handlers/HandlerAdd.java

```diff
@@ -45,14 +50,29 @@
     }
 
     @Override
+    protected void populateModel(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
+        super.populateModel(context, operation, resource);
+        final PathAddress address = PathAddress.pathAddress(operation.get(OP_ADDR));
+        final String name = address.getLastElement().getValue();
```

Comment:
Actually, why not just pass Handler.CAPABILITY to the superclass constructor? This seems to be just be doing what it would do in recordCapabilitiesAndRequirements(...).